lib/CMakeLists.txt

```diff
@@ -107,6 +112,14 @@ foreach(UTIL_LIB_NAME IN ITEMS Utils UtilsCpp)
   install(
     FILES ${PROJECT_BINARY_DIR}/${CMAKE_INSTALL_DATADIR}/cmake/OpenCL${UTIL_LIB_NAME}/OpenCL${UTIL_LIB_NAME}ConfigVersion.cmake
     DESTINATION ${config_package_location}
+    COMPONENT binary
+  )
+
+  # For packaging the utils libraries
+  install(
+    TARGETS ${UTIL_LIB_TARGET}
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    COMPONENT binary
```

Comment:
why was it added here?

The library is already installed above via:
```
 install(
    TARGETS ${UTIL_LIB_TARGET}
    EXPORT OpenCL${UTIL_LIB_NAME}Targets
    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
    COMPONENT binary
  )
```

installation w/o ARCHIVE LIBRARY or RUNTIME is bad practice